app/src/main/kotlin/com/wire/android/navigation/NavigationUtils.kt

```diff
@@ -23,21 +23,37 @@ internal fun navigateToItem(
                     }
                 }
             }
-            BackStackMode.NONE -> {}
+            BackStackMode.REMOVE_CURRENT -> {
+                navController.run {
+                    println("cyka 0 ${backQueue.map { it.destination.route }}")
```

Comment:
Do we need this log? I we do, please use the appLogger.

Reply:
removed it, sorry ;P